[BUG] Builds requires origin/main

**What is the bug?**

Scripts are referencing another branch, specifically origin/main.

```
testbox ~/source/opensearch-project/performance-analyzer/upstream-opensearch-project$ git init
Initialized empty Git repository in /home/ubuntu/source/opensearch-project/performance-analyzer/upstream-opensearch-project/.git/
testbox ~/source/opensearch-project/performance-analyzer/upstream-opensearch-project (master)$ git remote add origin https://github.com/opensearch-project/performance-analyzer.git
testbox ~/source/opensearch-project/performance-analyzer/upstream-opensearch-project (master)$ git fetch --depth 1 origin 1.0.0.0
remote: Enumerating objects: 280, done.
remote: Counting objects: 100% (280/280), done.
remote: Compressing objects: 100% (176/176), done.
remote: Total 280 (delta 88), reused 185 (delta 59), pack-reused 0
Receiving objects: 100% (280/280), 1.54 MiB | 5.63 MiB/s, done.
Resolving deltas: 100% (88/88), done.
From https://github.com/opensearch-project/performance-analyzer
 * tag               1.0.0.0    -> FETCH_HEAD
testbox ~/source/opensearch-project/performance-analyzer/upstream-opensearch-project (master)$ git checkout FETCH_HEAD
HEAD is now at 8bad5d2 Update OS upstream version and add release notes for 1.0.0.0 release (#32)
```

```
./gradlew build

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':spotlessApply'.
> Could not create task ':spotlessJavaApply'.
   > Could not create task ':spotlessJava'.
      > No such reference 'origin/main'
```

--- a/build.gradle
+++ b/build.gradle
@@ -88,9 +88,6 @@ ext {
 
 spotless {
     java {
-        // only apply formatting rules to updated files
-        ratchetFrom "origin/${gitDefaultBranch}"
-
         licenseHeaderFile(file('license-header'))
         googleJavaFormat().aosp()
         importOrder()

